packages/build-tools/src/builders/android.ts

```diff
@@ -105,7 +106,12 @@ async function buildAsync(ctx: BuildContext<Android.Job>): Promise<void> {
       gradleCommand,
       androidDir: path.join(ctx.getReactNativeProjectDirectory(), 'android'),
       ...(resolvedExpoUpdatesRuntimeVersion
-        ? { extraEnv: { EXPO_UPDATES_FINGERPRINT_OVERRIDE: resolvedExpoUpdatesRuntimeVersion } }
+        ? {
+            extraEnv: {
+              EXPO_UPDATES_FINGERPRINT_OVERRIDE: resolvedExpoUpdatesRuntimeVersion,
+              EXPO_UPDATES_WORKFLOW_OVERRIDE: ctx.job.type,
```

Comment:
this isn't technically needed since this is only used if `EXPO_UPDATES_FINGERPRINT_OVERRIDE` isn't provided, but worth adding for consistency just in case someone puts up a project that uses it.